src/Handlers/GeneralUiStateHandler.php

```diff
@@ -130,18 +144,17 @@ protected function _getWebsiteTimezone()
     }
 
     /**
-     * Get currency for website.
+     * Prepare application's currency configuration for state.
      *
      * @since [*next-version*]
      *
-     * @return array Website currency information.
+     * @param array|Traversable|stdClass $currencyConfig Currency config of application.
+     *
+     * @return array Application currency config.
      */
-    protected function _getWebsiteCurrency()
+    protected function _prepareCurrencyConfig($currencyConfig)
     {
-        return [
-            'name'   => edd_get_currency(),
-            'symbol' => edd_currency_symbol(),
-        ];
+        return $this->_normalizeArray($currencyConfig);
```

Comment:
In the future, this and other operations that convert configuration to array should apply *recursive* conversion.